lib/services/insights/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "azure-mgmt-insights",
+  "author": "Microsoft Corporation",
+  "contributors": [
+    "Calderon-Meza, Guillermo <gucalder@microsoft.com>"
```

Comment:
Nobody else is on this for the moment, but I can add them all just in case.
Done, I added them all.